libcalico-go/lib/backend/k8s/k8s_test.go

```diff
@@ -770,7 +770,7 @@ var _ = testutils.E2eDatastoreDescribe("Test Syncer API for Kubernetes backend",
 
 		gnpClient := c.GetResourceClientFromResourceKind(apiv3.KindGlobalNetworkPolicy)
 		kvp1Name := "my-test-gnp"
-		kvp1KeyV1 := model.PolicyKey{Name: kvp1Name}
+		kvp1KeyV1 := model.PolicyKey{Name: kvp1Name, Tier: "default"}
```

Comment:
Don't we have any tests for CRUD on tiers?

Reply:
apparently, we don't! I'll add it.